src/core/jsdoc-tsd-parser.ts

```diff
@@ -91,6 +103,7 @@ export class JSDocTsdParser {
 		return domParams;
 	}
 
+	// tslint:disable-next-line:member-ordering

```

Comment:
Maybe we should create the property declaration with the dts-dom-module like dom.create.property(...). It's more safe if the API changes in future..